docs/typedoc/classes/authorizationapiclient.html

```diff
@@ -0,0 +1,229 @@
+<!doctype html>
```

Comment:
@nguyentoanit we should not commit generated docs

Reply:
I removed them from ci: https://github.com/ScaleLeap/selling-partner-api-sdk/pull/147/commits/de11111049633d9bddd951b8b42e8dee4c25dea9